werkit/common/pathlib.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+def project_relative_path(*path_components):
```

Comment:
This is unused

Reply:
Fixed in 6cc76cfc70e885258c5423fc488f9284a09dfd56